### DISCUSSION TITLE
Unmanage all window by default

### DISCUSSION THREAD

**Discussion by Inc0n** [asmvik/yabai#1000] [Q&A] [answered]
Is it possible to unmanage all applications by default, and then I could configure which ones I want to be managed?

**Comment by jasonvarga** [comment]
I would find this handy too.

I use a single bsp space, but there are a lot of applications that I would prefer to have floating. I have to maintain a long list of them in my yabairc file. I think the list would be shorter if I could opt into management.

**Comment by Inc0n** [comment, marked as answer]
As per the [rules documentation](https://github.com/koekeishiya/yabai/blob/master/doc/yabai.asciidoc#rule), you can have the following line to achieve this:
Here I want just Emacs and Firefox to be managed  
```
yabai -m rule --add app!="Emacs|Firefox" manage=off
```

**Reply by jasonvarga** [reply]
Works well, thank you.

**Reply by Inc0n** [reply]
My addition would be to add a grid configuration `grid=6:6:1:1:4:5` (read the doc and change and experiment with the numbers to find the best for you) to the rule, since now all the unmanaged windows are relocated at the top of either side of the screen.

**Reply by Inc0n** [reply]
I am also encountering issues with notifications now as well, they are shifted to the left and only half of the body is shown...
